src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs

```diff
@@ -16,7 +16,9 @@ public abstract class AzureEmailProviderBase : IEmailProvider
     // Common supported file extensions and their corresponding MIME types for email attachments
     // using Azure Communication Services Email.
     // For more info <see href="https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-attachment-allowed-mime-types" />
+#pragma warning disable CA1707 // Remove the underscores from member name
     protected static readonly Dictionary<string, string> _allowedMimeTypes = new()
```

Comment:
This should actually be `AllowedMimeTypes` anyway.

Reply:
It's OK as a breaking change to change this?

Reply:
Yes, because this code wasn't released yet.

Reply:
pushed a commit to change the name to be upper-case